src/components/CategoryList/index.test.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import 'test-util/setup';
+import CategoryList from './index';
+
+const itemRenderer = ({ data = {} }) => (
+  <div className="item-rendered">{data.description}</div>
```

Comment:
i'd like to see tests for clicks, is-open and itemRenderer

Reply:
done